ui/jobs/components/drk.ts

```diff
@@ -1,12 +1,14 @@
+import { Bars } from '../bar';
 import { kAbility } from '../constants';
 import { computeBackgroundColorFrom } from '../utils';
+import { JobDetail } from '../../../types/event';
 
-let resetFunc = null;
-let tid1;
-let tid2;
-let tid3;
+let resetFunc: ((bars: Bars) => void) = (_bars: Bars) => undefined;
+let tid1: NodeJS.Timeout;
```

Comment:
I think this is not a `NodeJS.Timeout` as it is not running inside of NodeJS, unless I am misunderstanding something about this type? I think this is just a `number`.

Reply:
even through `clearTimeout` receive a `number`, bot `setTimeout` return a `NodeJS.Timeout` which is not a super type of `number`

Reply:
try to add `window.` prefix to the `setTimeout` function then TypeScript knows that is the `setTimeout` function from browser rather than the one from `Node.js`.